src/herfindahlIndex.js

```diff
@@ -0,0 +1,24 @@
+var enforceNumber = require('./utils/enforce/number');
+
+/**
+ * Calculates Herfindahl-Hirschman Index, a measure of market concentration and competition, used by U.S. governments for antitrust enforcement.
+ * @param {array} marketShare (expressed as percentage value for each firm)
+ * @returns {number} Herfindahl-Hirschman Index (HHI), a number between 0 and 10,000
+ */
+
+function herfindahlIndex(marketShare) {
+  var result = 0;
+
+  enforceNumber(marketShare);
```

Comment:
Delete this line. `enforceNumber` is used to check that an argument is a number. In this case we want to ensure it's an array. And the fact that it's not throwing an error indicates it's completely broken...